src/components/NBadge/NBadge.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <transition name="nitrozen-badge">
+    <div class="nitrozen-badge" :class="[addClass]">{{title.toUpperCase()}}</div>
+  </transition>
+</template>
+
+<script>
+export default {
+  name: "nitrozen-badge",
+  props: {
+    state: {
+      type: String,
+      default: "none"
+    },
+    title: {
```

Comment:
vue slots can be used instaed of title prop

https://vuejs.org/v2/guide/components-slots.html